docs/beds/dewertokin.md

```
@@ -175,12 +175,22 @@ From app disassembly analysis (FurniMove):
 
-Some DewertOkin devices use an RF Gateway with an 8-byte protocol:
+Some DewertOkin devices identify through the Device Information Service as
+manufacturer `DewertOkin GmbH`, model `Bluetooth RF-Gateway`. These receivers
+can expose the normal `62741523-...` / `62741525-...` Okin service, but motor
+commands use a separate RF-Gateway service instead:
+
+- **Service:** `92111420-72ab-4564-62ef-2a881286a6b0`
+- **Write Characteristic:** `92111422-72ab-4564-62ef-2a881286a6b0`
+
+The RF-Gateway command format is 8 bytes:
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)